.github/workflows/test.yaml

```diff
@@ -76,6 +76,8 @@ jobs:
   oasdiff_breaking:
     runs-on: ubuntu-latest
     name: Test breaking changes
+    env:
+      EXPECTED_OUTPUT: "1 changes: 1 error, 0 warning, 0 info"
```

Comment:
Lets add a prefix to our environment variables, like: `OASDIFF_ACTION_TEST_EXPECTED_OUTPUT`

Reply:
Done